### DISCUSSION TITLE
ask for remove processor and ram restriction

### DISCUSSION THREAD

**Discussion by yosouvick** [MustardChef/WSABuilds#270] [General]
hello...i am asking you to remove processor and ram restriction.i have an unsupported pc and cannot install this build on my pc

**Comment by cody8764** [comment]
Your PC will crash if you do so.

These restrictions are for your own good.

My PC which is supported actually kinda struggles to run WSA.

![image](https://github.com/MustardChef/WSABuilds/assets/111713735/8c479595-cf5c-4e28-941b-25fc230b15df)

Specs of my PC

11th gen intel core i5 1135g7
12GB ram 11.7GB usable
512GB SSD
intel iris xe graphics iGPU

**Reply by cody8764** [reply]
I'm just a regular user of these wsa builds.

**Reply by MustardChef** [reply]
You can restrict WSA's RAM usage from its settings. By default it is set to `6GB (Default)` for my PC (it may be different for your PC). However, you can set it down to ``2GB (Low)``.

**Comment by MustardChef** [comment]
> hello...i am asking you to remove processor and ram restriction.i have an unsupported pc and cannot install this build on my pc

You must have 8GB of RAM. It is possible to run WSA on unsupported CPU, but your milage and success will vary on what CPU you have.
